README.md

```diff
@@ -52,21 +54,41 @@ The HelloID connector uses the API endpoints listed in the table below.
-| delete.ps1           | Delete a user account                                  | Be careful when implementing this! There is no way to restore deleted users. |
-| permissions.ps1      | Retrieves all groups and provides them as entitlements |                                                                              |
-| grantPermission.ps1  | Add a user account to a group                          |                                                                              |
-| revokePermission.ps1 | Remove a user account from a group                     |                                                                              |
-| revokePermission.ps1 | Remove a user account from a group                     |                                                                              |
-| resourceCreation.ps1 | Create a group for provided resource, e.g. department  |                                                                              |
+| Action                 | Action(s) Performed                                    | Comment                                                                      |
+| ---------------------- | ------------------------------------------------------ | ---------------------------------------------------------------------------- |
+| create.ps1             | Create (or update) and correlate a user account.       |                                                                              |
+| enable.ps1             | Enable a user account                                  |                                                                              |
+| update.ps1             | Update a user account                                  |                                                                              |
+| disable.ps1            | Disable a user account                                 |                                                                              |
+| delete.ps1             | Delete a user account                                  | Be careful when implementing this! There is no way to restore deleted users. |
+| permissions.ps1        | Retrieves all groups and provides them as entitlements |                                                                              |
+| grantPermission.ps1    | Add a user account to a group                          |                                                                              |
+| revokePermission.ps1   | Remove a user account from a group                     |                                                                              |
+| dynamicPermissions.ps1 | Add/remove a user account to/from a group              |                                                                              |
+| resourceCreation.ps1   | Create a group for provided resource, e.g. department  |                                                                              |
+
+### Mapping
```

Comment:
What I miss in the mapping overview are the following:
- type (text or array)
- checkbox use in notifications
Is there also a possibility to make sure the table fits in the default available space? Now there is a scrollbar to see the content on the right side of the table.

Reply:
Added the type and checkbox use for notifications. The use for notifications is of course entirely based on preference, but the default is no for all fields.
As for the possibility to make it fit to the available space, sadly we have no control over this. We could use CSS to set specifics, but I don't want to get into that as this part of the mapping should only be temporary until we have the import/export option.